app/src/main/res/values/strings.xml

```diff
@@ -927,12 +927,12 @@ Do you want to switch to this account?</string>
   <string name="self_hosted_server_url">Self-hosted server URL</string>
   <string name="passkey_operation_failed_because_user_could_not_be_verified">Passkey operation failed because user could not be verified.</string>
   <string name="user_verification_direction">User verification</string>
-  <string name="welcome_title_1">Privacy, prioritized</string>
+  <string name="privacy_protected">Privacy, prioritized</string>
```

Comment:
Should this be `privacy_ prioritized`?

Reply:
It absolutely should be, and I should get some caffeine! 👍